tools/summarise-ct-results

```diff
@@ -85,9 +86,12 @@ add_up_suite_results(SuiteSummaries) ->
                 end, {0, 0, 0, 0}, SuiteSummaries).
 
 add_up_group_results(GroupSummaries) ->
-    lists:foldl(fun(Filename, {OkAcc, FAcc, EvOkAcc}) ->
+    lists:foldl(fun(Filename, {OkAcc, FAcc, EvOkAcc, EndPerTestcaseFailedAcc}) ->
                         {ok, Terms} = file:consult(Filename),
+                        Map = maps:from_list(Terms),
                         #{eventually_ok_tests := EvOk,
-                          groups_summary := {Ok, F}} = maps:from_list(Terms),
-                        {OkAcc + Ok, FAcc + F, EvOkAcc + EvOk}
-                end, {0, 0, 0}, GroupSummaries).
+                          groups_summary := {Ok, F}}
+                            = Map,
```

Comment:
Why two lines?

Reply:
To have compatibility with files in the old format (where `end_per_testcase_failures` is not in the map)